### ISSUE TITLE
I don't find my language

### ISSUE THREAD

**Issue by lordm121** [matheuss/google-translate-api#124]
I think google added Kurdish language but this npm not added yet

**Comment by AidanWelch** [comment]
Yes this package is no longer being updated, I am maintaining a fork that is, and am adding support for all new languages soon: [here](https://www.npmjs.com/package/google-translate-api-x)
